API-6263: Add `subtype` property to greetings (#714)
livechat/livechat-public-docs: content/management/configuration-api/v2.0/index.mdx

| -------- | -------------------------------------------------------------------------|
| `group`  | creates a greeting displayed in a particular group only. Defaults to `0` |
| `active` | creates a greeting, which is active or inactive. Defaults to `1`          |

> | `subtype` | possible values: `announcement`; you can have up to 10 `announcement` greetings; if given `announcement` greeting is canceled by the customer, it'll never be shown again for that customer |

Reviewer: ```suggestion
| `subtype` | possible values: `announcement`; you can have up to 10 `announcement` greetings; if a given `announcement` greeting is canceled by a customer, it'll never be displayed to that customer again |
```
